tests/test_nsjail.py

```diff
@@ -135,25 +223,27 @@ def test_sigsegv_returns_139(self):  # In honour of Juan.
             """
         ).strip()
 
-        result = self.nsjail.python3(code)
+        result = self.eval_file(code)
         self.assertEqual(result.returncode, 139)
         self.assertEqual(result.stdout, "")
         self.assertEqual(result.stderr, None)
 
     def test_null_byte_value_error(self):
-        result = self.nsjail.python3("\0")
+        # This error does not occur without -c, where it
+        # would be a normal SyntaxError.
+        result = self.nsjail.python3(["-c", "\0"])
```

Comment:
This can use the `eval_code` utility, right?

Reply:
Yup, using `eval_code` with [fca97ce](https://github.com/python-discord/snekbox/pull/159/commits/fca97cecea5040dac8ab02046cffac0713f78dd2)